Client/Frontend/Widgets/Menu/MenuView.swift

```diff
@@ -223,6 +225,19 @@ class MenuView: UIView {
         toolbarButtonSelected(view)
     }
 
+    @objc private func menuLongPressed(recognizer: UILongPressGestureRecognizer) {
+        guard recognizer.state == .Began else {
+            return
+        }
+        let collectionView: UICollectionView = self.menuPagingView
```

Comment:
not sure what the point of assigning this to another variable is. Are you trying to make it explicit that the `menuPagingView` is a collectionView for clarity?